pkg/telemetry/phonehome/telemeter.go

```diff
@@ -1,34 +1,14 @@
 package phonehome
 
-// Telemeter defines a common interface for telemetry gatherers.
```

Comment:
nitpick: I think this change could, in theory, be reverted now, right?

Reply:
Do you mean extracting the interface in a dedicated package? No, it cannot be reverted.